src/snowflake/connector/cache.py

```diff
@@ -572,6 +598,10 @@ def _save_if_should(self) -> bool:
             return self._save()
         return False
 
+    def save(self, load_first: bool = True):
```

Comment:
I don't think that we should expose this

Reply:
I will remove this